### PR TITLE
Project setup

### PR DESCRIPTION
**Pull request by Martum**
### Summary
Add Rails initial setup.

### Trello card
https://trello.com/c/cPDyeeBD/16-crear-aplicacion-rails-wbooksapi

### CODE REVIEW

File: config/database.yml [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+#   DATABASE_URL="postgres://myuser:mypass@localhost/somedatabase"
+#
+# You can use this database configuration with:
+#
+#   production:
+#     url: <%= ENV['DATABASE_URL'] %>
+#
+production:
+  <<: *default
+  database: mm-rails_production
```
Comment (francoprud) [inline config/database.yml]: Also add the following:

```yml
username: wbooksapi
password: <%= ENV['WBOOKS-API_DATABASE_PASSWORD'] %>
```

File: config/database.yml [hunk 2]
```diff
@@ -0,0 +1,86 @@
+# PostgreSQL. Versions 9.1 and up are supported.
```
Comment (francoprud) [inline config/database.yml]: Remove this pg comments

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+# If you find yourself ignoring temporary files generated by your text editor
+# or operating system, you probably want to add a global ignore instead:
+#   git config --global core.excludesfile '~/.gitignore_global'
+
+# Ignore bundler config.
+/.bundle
+
+# Ignore all logfiles and tempfiles.
+/log/*
+/tmp/*
```
Comment (francoprud) [inline .gitignore]: Ignore the complete folder with `/tmp`

File: .gitignore [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+!/log/.keep
+!/tmp/.keep
```
Comment (francoprud) [inline .gitignore]: What is this?

Comment (Martum) [inline .gitignore]: Its the RoR default `.gitignore` :stuck_out_tongue: I will remove those lines.

Comment (francoprud) [inline .gitignore]: Nice!

**Pull request merged**